.github/workflows/test-pytest.yml

```diff
@@ -10,7 +10,7 @@ on:
 jobs:
   test:
     strategy:
-      fail-fast: true
+      fail-fast: false
```

Comment:
i know we touched on this, but to confirm, is this just for now or more generally going forward? once we've got the conditional solution is there any reason we wouldn't revert to `fail-fast`?